src/ingest_llm_as/services/llm_cache.py

```
@@ -193,164 +196,171 @@ async def set(
+
                     try:
                         entry_dict = json.loads(cached_data)
-                        timestamp = datetime.fromisoformat(entry_dict['timestamp'])
-                        
+                        timestamp = datetime.fromisoformat(entry_dict["timestamp"])
+
                         if oldest_entry is None or timestamp < oldest_entry:
                             oldest_entry = timestamp
                         if newest_entry is None or timestamp > newest_entry:
                             newest_entry = timestamp
                     except:
```

Ruff (0.13.3)

Do not use bare `except`

(E722)